src/backend/kvm/thread.rs

```diff
@@ -117,7 +120,42 @@ impl<P: KeepPersonality> Thread<P> {
                 };
                 Ok(None)
             }
+            item::Enarxcall {
+                num: item::enarxcall::Number::Park,
+                argv: [val, timeout, ..],
+                ret,
+                ..
+            } => {
+                let timeout = if *timeout != sallyport::NULL {
+                    Some(unsafe {
+                        // Safety: `deref_aligned` gives us a pointer to an aligned `timespec` struct.
+                        // We also know, that the resulting pointer is inside the allocated sallyport block, where `data`
+                        // is a subslice of.
+                        (*deref_aligned::<MaybeUninit<timespec>>(data, *timeout, 1)
```

Comment:
Why `MaybeUninit`? Isn't it redundant if you call `assume_init` right after?

Reply:
fixed